src/empty-state-card.tsx

```diff
@@ -70,24 +70,32 @@ const EmptyStateCard: React.FC<Props> = ({
         image = <img className={classes.image} alt="" src={imageUrl} />;
     }
 
+    let resizedIcon;
+    if (icon) {
+        resizedIcon = React.cloneElement(icon, {size: isTabletOrSmaller ? 64 : 80, color: colors.brand});
```

Comment:
I don't really like this, you're assuming here that you will receive an Icon component with a size prop, but this component accepts anything as an icon (it could be, for example, an <img>)

Reply:
Ok, In this case, I'll  revert the change so the size won't be controlled from the component, and we will expect developers to use the correct size

Reply:
perhaps a better option would be to set the size of the container and assume that the developer will use an asset with 100% width (or the correct size)